src/Bicep.Core.Samples/Files/ResourcesManagementGroup_CRLF/main.json

```diff
@@ -62,6 +62,25 @@
         "[format('Microsoft.Authorization/roleAssignments/{0}', guid('contributor', parameters('contributorPrincipals')[0]))]",
         "[format('Microsoft.Authorization/roleAssignments/{0}', guid('owner', parameters('ownerPrincipalId')))]"
       ]
+    },
+    {
+      "type": "Microsoft.Management/managementGroups",
```

Comment:
>managementGroups [](http://example.com/codeflow?start=36&length=16)

I don't have access to an enrollment account to try creating subs, but what-if was ok with the MG being created from RG scope.